tests/openvino/test_exporters_cli.py

```diff
@@ -253,10 +253,12 @@ def test_exporters_cli_int4(self, task: str, model_type: str, option: str, expec
 
     def test_exporters_cli_int4_with_local_model_and_default_config(self):
         with TemporaryDirectory() as tmpdir:
-            pt_model = AutoModelForCausalLM.from_pretrained(MODEL_NAMES["falcon-40b"])
-            # overload for matching with default configuration
-            pt_model.config._name_or_path = "tiiuae/falcon-7b-instruct"
+            model_id = "bigscience/bloomz-560m"
```

Comment:
I think we can keep using `MODEL_NAMES["falcon-40b"]` which should be much smaller no ?

Reply:
done